api/util/mappers/sdk.py

```diff
@@ -0,0 +1,51 @@
+from typing import TYPE_CHECKING, TypeAlias
+
+from util.mappers.engine import (
+    map_environment_to_engine,
+    map_identity_to_engine,
+)
+
+if TYPE_CHECKING:
+    from environments.models import Environment
+
+ENVIRONMENT_RESPONSE_EXCLUDE_FIELDS = [
+    "amplitude_config",
+    "dynatrace_config",
+    "heap_config",
+    "mixpanel_config",
+    "rudderstack_config",
+    "segment_config",
+    "webhook_config",
+]
+
+
+SDKDocumentValue: TypeAlias = dict[str, "SDKDocumentValue"] | str | bool | None | float
+SDKDocument: TypeAlias = dict[str, SDKDocumentValue]
+
+
+def map_environment_to_sdk_document(
+    environment: "Environment",
+) -> SDKDocument:
+    """
+    Map an `environments.models.Environment` instance to an SDK document
+    used by SDKs with local evaluation mode.
+
+    It's virtually the same data that gets indexed in DynamoDB,
+    except it presents identity overrides and omits integrations configurations.
```

Comment:
Intrigued to understand which part of the below actually omits the integrations configurations?

Reply:
See https://github.com/Flagsmith/flagsmith/blob/ade08b0f6f84b0d27d96f1602d3ec0518b7ad578/api/util/mappers/sdk.py#L50-L51.

Reply:
Blimey, yep, not sure how I missed that 🤦